src/torch/layer.ml

```diff
@@ -221,17 +449,60 @@ let layer_norm vs ?(cudnn_enable = true) ?(eps = 1e-5) dim =
   let weight = Var_store.new_var vs ~name:"weight" ~shape:[ dim ] ~init:Ones in
   let bias = Var_store.new_var vs ~name:"bias" ~shape:[ dim ] ~init:Zeros in
   let apply xs =
-    Tensor.layer_norm
+    Tensor.layer_norm2d
       xs
-      ~normalized_shape:[ dim ]
+      ~num_features:[ dim ]
       ~weight:(Some weight)
       ~bias:(Some bias)
       ~eps
+      ~momentum
+
       ~cudnn_enable
   in
   { apply }
 ;;
 
+let instance_norm2d
```

Comment:
I'm going to omit instance norm because
* I think the C++ API torch::instance_norm might be arbitrary-dimensional; not a 2D version. Would need to dig into the code since it's not obviously documented.
* There's no training parameter.
* If we define one of the instance_norm* functions, I think we should define all of them